pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/BaseOp.java

```diff
@@ -65,6 +66,22 @@ public String getDescription() {
     return _description;
   }
 
+  public String getParentDir() {
+    return _parentDir;
+  }
+
+  public void setParentDir(String parentDir) {
+    _parentDir = parentDir;
+  }
+
+  public String getAbsoluteFileName(String fileName) {
+    //change if the file name is relative
+    if(fileName.startsWith("/")) {
```

Comment:
file name will always start with "/". Make sure of it in the shell script, like I suggested last time.

Reply:
updated the script to pass absolute path